index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt(`Ingrese nombre`);
+alert(`Hola mundo`);
```

Comment:
En este caso no dice el nombre, dice mundo.


```suggestion
alert(`Hola ${nombreUsuario}!`);
```